# Bioinformatics-class-project-GloVe

## General reference:

1. Data is from [Predictive metabolomic profiling of microbial communities using amplicon or metagenomic sequences](https://www.nature.com/articles/s41467-019-10927-1#MOESM1)

2. GloVe embedding for abundance table can refer [Decoding the Language of Microbiomes using Word-Embedding Techniques, and Applications in Inflammatory Bowel Disease](https://www.biorxiv.org/content/10.1101/748152v1.full.pdf)

3. General GloVe embedding tutorial can be found in [GloVe: Global Vectors for Word Representation](https://nlp.stanford.edu/projects/glove/)

4. Some code example: [GloVe embedding demo](https://github.com/stanfordnlp/GloVe/blob/master/demo.sh)

## Data explanation:

1. abundance_table.csv: species frequence per sample
2. metabolite_table.csv: metabolite concentration per sample

Both tables have labels (CD, UC and Control) and sample id that ties species frequence and metabolite concentration together.

## Overall goal:

** you will need to install GloVe software for this project **

Apply GLoVe embedding to study the metabolite and species association to Crohnâ€™s disease

The students working on this project should come up with detailed research directions and delieverables.
